docs/platform/known-issues.md

```diff
@@ -0,0 +1,36 @@
+---
+description: known issue in the Grid Platform
+---
+
+# Known Issues
+#### Last Updated: April 1, 2022
+
+### [BYOC](https://docs.grid.ai/platform/custom-cloud-credentials/byoc)
+* Creating runs with the same name across two cluster is currenly supported, but prohibits certain operations against runs/experiments. 
```

Comment:
what specific operations are prohibited?

Reply:
@oojo12 Can you add more information for this ??